src/handlers/handlers.test.ts

```diff
@@ -172,6 +195,8 @@ describe("Handlers", () => {
 
       const queue = await waitForEventToBeEmitted(clientSocket, "lobby:list");
 
+      expect(courtServiceLeaveMock).toHaveBeenCalledWith(serverSocket?.id);
```

Comment:
I expected to see some tests failing, then i noticed it was missing mocking the service for this handler